### ISSUE TITLE
Crunz sends output email even if the output is empty

### ISSUE THREAD

**Issue by kinoute** [lavary/crunz#64]
Hi,

I was using traditional cronjobs with a MAILTO variable and some other args to receive mails only when my scripts output something (i.e. not empty).

I switched to crunz and tried to reproduce the same thing. Unfortunately, I can't. I receive emails for every cronjob even if the output of the scripts is empty. I think crunz should only send an output (when we activate them in the conf file) if there is something to send. If it's empty, nothing should be mailed.

Am I missing something?

Regards

**Comment by PabloKowalczyk** [comment]
Hello, do you still face this issue?

**Comment by kinoute** [comment]
Yes. I just tried.

Steps to reproduce:

1) set "email_output" in the config file to "true" and add emails infos into the "mailer" part
2) create a php file with a job that should output nothing like:

```
<?php
for ($i = 1; $i <= 5; $i++) {
    sleep(1);
}
```
3) Create the Tasks file:

```
<?php

use Crunz\Schedule;

$schedule = new Schedule();

$schedule->run('/usr/bin/php loop.php')       
         ->everyMinute()
         ->description('Testing a short loop');
         
return $schedule;

```

4) run `./vendor/bin/crunz schedule:run`

I received an email with no real output (just the description & the filename):

![capture d ecran 2018-05-11 a 16 52 34](https://user-images.githubusercontent.com/623763/39930721-c1eb910a-553b-11e8-828a-fa7f060e15cc.png)

If I add an echo in my loop, the result is normal and I get the output:

![capture d ecran 2018-05-11 a 16 53 26](https://user-images.githubusercontent.com/623763/39930756-d97d6334-553b-11e8-9dc5-02a07ac39191.png)

I don't think crunz should trigger an email when nothing is present in the output, that's why I added the use of "empty" in my PR. An empty line between the description/filename & the real output would be nice too I guess.
